### PR TITLE
Reduce the waiting time for ready in e2e

### LINKED ISSUE #315: e2e test:clusterpedia installation failure does not terminate the test
### What happened?

In e2e testing, clusterpedia installation failure does not terminate the test

https://github.com/clusterpedia-io/clusterpedia/runs/7849779721?check_suite_focus=true#step:4:1860

### What did you expect to happen?

If an exception occurs during clusterpedia installation, then terminate the e2e test

### How can we reproduce it (as minimally and precisely as possible)?

None

### Anything else we need to know?

_No response_

### Clusterpedia Version

<details>

```console
# replace the pod name of clusterpedia apiserver
$ kubectl -n clusterpedia-system exec {clusterpedia-apiserver-pod-name} -- apiserver --version=raw
# paste output here

# replace the pod name of clusterpedia clustersynchro-manager
$ kubectl -n clusterpedia-system exec {clustersynchro-manager-pod-name} -- clustersynchro-manager --version=raw
# paste output here
```

</details>

### Host Kubernetes version

<details>

```console
$ kubectl version
# paste output here
```

</details>

### PR DESCRIPTION
**Pull request by wzshiming**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #315

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Iceber** [comment]
This does not seem to be the same problem as the issue, which proposes to terminate the test if the clusterpedia installation fails

https://github.com/clusterpedia-io/clusterpedia/runs/7849779721?check_suite_focus=true#step:4:1860
>   customresourcedefinition.apiextensions.k8s.io/clustersyncresources.cluster.clusterpedia.io created
  customresourcedefinition.apiextensions.k8s.io/pediaclusters.cluster.clusterpedia.io created
  customresourcedefinition.apiextensions.k8s.io/clusterimportpolicies.policy.clusterpedia.io created
  customresourcedefinition.apiextensions.k8s.io/pediaclusterlifecycles.policy.clusterpedia.io created
  "bitnami" already exists with the same configuration, skipping
  Error: the lock file (Chart.lock) is out of sync with the dependencies file (Chart.yaml). Please update the dependencies
  Error: INSTALLATION FAILED: An error occurred while checking for chart dependencies. You may need to run `helm dependency build` to fetch missing dependencies: found in Chart.yaml, but missing in charts/ directory: postgresql, mysql, common
  kubectl get all -n clusterpedia-system
  No resources found in clusterpedia-system namespace.

clusterpedia installation failed, but e2e continues to run

**Comment by wzshiming** [comment]
> This does not seem to be the same problem as the issue, which proposes to terminate the test if the clusterpedia installation fails

https://github.com/clusterpedia-io/clusterpedia/pull/311 Due to its modifications that prevented the installation from succeeding, shortening the waiting time can be a unified check for this kind of problem

**Comment by Iceber** [comment]
Couldn't we just quit when the installation fails ? so that the logs can be terminated at the place of failure, and it's easy to troubleshoot.

File: test/cases/workable.test.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Waiting for Clusterpedia to be ready
 function ready() {
     local i
-    for ((i = 0; i < 300; i++)); do
+    for ((i = 0; i < 10; i++)); do
```
Comment (Iceber) [inline test/cases/workable.test.sh:6]: github action performance is not very good, from 300 directly down to 10, whether it becomes prone to failure.

Let's go down to 200 or 150 first?

**Comment by wzshiming** [comment]
Updated, take a look again, thx.

**Pull request merged**